### PR TITLE
updated magento documentation, added idea folder to gitignore

### PR DESCRIPTION
**Pull request by sergy-gzh**

### CODE REVIEW

**Comment by cane-lagerwaard** [comment]
I think it would be nice to introduce docs versioning here https://docusaurus.io/docs/en/versioning so old API is still available, or we could maybe add tabs instead?
What do you think @Borales @mateuszwozniak

File: docs/backend/installing-magento2.md [hunk 1]
```diff
@@ -65,38 +65,48 @@ Please be aware that Magento performance depends heavily on which mode it is run
...
-- `[PUT] /rest/V1/carts/mine/deity-order` - place order with Adyen credit card as a logged in customer - getting an object as a response
-- `[PUT] /rest/V1/guest-carts/:cartId/deity-order` - place order with Adyen credit card - getting an object as a response
-- `[PUT] /rest/V1/customers/me/address` - update customer address
-- `[PUT] /rest/V1/customers/me/newsletter/subscribe` - subscribe customer to newsletter
-- `[PUT] /rest/V1/customers/me/newsletter/unsubscribe` - unsubscribe customer to newsletter
-- `[DELETE] /rest/V1/customers/me/address/:addressId` - remove customer address
+Custom REST API endpoints provided by this module:
+
+### General
+- `[GET] /V1/falcon/urls` - Url dispatch. Provides info about entity behind the url or error if URL does not exist in magento.
```
Comment (Borales) [inline docs/backend/installing-magento2.md]:
```suggestion
- `[GET] /V1/falcon/urls` - Url resolver. Provides info about entity behind the url or error if URL does not exist in Magento.
```

Not really sure if "dispatch" is a correct word in this case

File: docs/backend/installing-magento2.md [hunk 2]
```diff
@@ -65,38 +65,48 @@ Please be aware that Magento performance depends heavily on which mode it is run
...
+
+### Catalog
+
+- `[GET] /V1/falcon/breadcrumbs` - Get list of breadcrumbs for given url.
+- `[GET] /V1/falcon/categories/:categoryId/products` - Get product and filters data for given category id. Can be used to provide filtered or sorted content.
+
+### Checkout
+- `[POST] /V1/falcon/carts/mine/save-payment-information-and-order` - Save payment information and place order for customer.
+- `[POST] /V1/falcon/guest-carts/:cartId/save-payment-information-and-order` - Save payment information and place order for guest.
+- `[PUT] /V1/falcon/carts/mine/place-order` - place order for customer.
```
Comment (Borales) [inline docs/backend/installing-magento2.md]:
```suggestion
- `[PUT] /V1/falcon/carts/mine/place-order` - Place order for customer.
```

(to keep the consistency)

File: docs/backend/installing-magento2.md [hunk 3]
```diff
@@ -65,38 +65,48 @@ Please be aware that Magento performance depends heavily on which mode it is run
...
+- `[PUT] /V1/falcon/guest-carts/:cartId/place-order` - place order for guest.
```
Comment (Borales) [inline docs/backend/installing-magento2.md]:
```suggestion
- `[PUT] /V1/falcon/guest-carts/:cartId/place-order` - Place order for guest.
```

File: docs/backend/installing-magento2.md [hunk 4]
```diff
@@ -65,38 +65,48 @@ Please be aware that Magento performance depends heavily on which mode it is run
...
+- `[POST] /V1/falcon/carts/mine` - Get existing or create customer cart id.
+
+### Customer
+
+- `[PUT] /V1/falcon/customers/password/reset` - Rest password API.
```
Comment (Borales) [inline docs/backend/installing-magento2.md]:
```suggestion
- `[PUT] /V1/falcon/customers/password/reset` - Reset password API.
```

File: docs/backend/installing-magento2.md [hunk 5]
```diff
@@ -65,38 +65,48 @@ Please be aware that Magento performance depends heavily on which mode it is run
...
+- `[GET] /V1/falcon/customers/me/address/` - Get list of customer addresses.
+- `[GET] /V1/falcon/customers/me/address/:addressId` - Get customer address info.
+- `[POST] /V1/falcon/customers/me/address` - Create customer address.
+- `[PUT] /V1/falcon/customers/me/address` - Updated customer address.
+- `[DELETE] /V1/falcon/customers/me/address/:addressId` - Delete customer address.
+- `[PUT] /V1/falcon/customers/me/newsletter/subscribe` - Subscribe customer.
```
Comment (Borales) [inline docs/backend/installing-magento2.md]:
```suggestion
- `[PUT] /V1/falcon/customers/me/newsletter/subscribe` - Subscribe customer to newsletter.
```

File: docs/backend/installing-magento2.md [hunk 6]
```diff
@@ -65,38 +65,48 @@ Please be aware that Magento performance depends heavily on which mode it is run
...
+- `[PUT] /V1/falcon/customers/me/newsletter/unsubscribe` - Unsubscribe customer.
```
Comment (Borales) [inline docs/backend/installing-magento2.md]:
```suggestion
- `[PUT] /V1/falcon/customers/me/newsletter/unsubscribe` - Unsubscribe customer from newsletter.
```

**Pull request merged**